eks-deployments/webhook/gmsa-webhook-template.yaml

```diff
@@ -76,7 +76,7 @@ spec:
         beta.kubernetes.io/os: linux
       containers:
       - name: ${NAME}
-        image: wk88/k8s-gmsa-webhook:${WEBHOOKIMAGE}
+        image: wk88/k8s-gmsa-webhook:latest
```

Comment:
Does `latest` work on all supported SKUs?

Reply:
Correct Sid, it works for all versions starting from 1.16